### DISCUSSION TITLE
Napalm username

### DISCUSSION THREAD

**Discussion by resparis** [netbox-community/netbox-docker#706] [Getting Help] [answered]
Hi All, 

I add the username and pass to netbox.env and I restarted the container:
NAPALM_PASSWORD=my_pass
NAPALM_TIMEOUT=10
NAPALM_USERNAME=my_user

I am getting ConnectAuthError.

I tried the API Call using postman but I did specify X-NAPALM-Username & X-NAPALM-Password in the headers and worked. 

Any ideas why the .env is not working?

**Comment by ryanmerolle** [comment, marked as answer]
Suggested steps to troubleshoot:

- Log into the container and printenv do see if it is set.
- Make sure you are using the netbox.env file in the env directory

**Reply by resparis** [reply]
I finally found the issue. After I updated the .env file, I was using "docker-compose up" instead of "docker-compose start"...
